predict.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
Does this have to be in this PR or can it be in a separate PR?

Reply:
Better to separate it out. I've removed it and will open another PR once this is merged.